### PR TITLE
Plangraph ui and perfomance fixes feature

### PR DESCRIPTION
**Pull request by remmintan**
Tasks PTSK0001630 and PTSK0001642

### CODE REVIEW

File: API/Tenders.API/Controllers/TenderPlanPositionController.cs [hunk 1]
```diff
@@ -72,6 +78,35 @@ public override ActionResult<ListResponse<TenderPlanPosition>> Get([FromQuery] F
             }
         }
 
+        /// <summary>
+        /// Метод получения списка позиций планов графиков для отображения на UI с возможностью фильтрации.
```
Comment (DamienVStone) [inline API/Tenders.API/Controllers/TenderPlanPositionController.cs]: Comments should be English

File: API/Tenders.API.DAL.Postgres/Services/PostgresRepository.cs [hunk 2]
```diff
@@ -57,7 +57,21 @@ public bool ChangeActiveFlag(string id, bool isActive)
         /// <inheritdoc/>
         public long Count(int skip, int take, string quickSearch = "", bool isActive = true)
         {
-            return this.Get(skip, take, quickSearch, isActive).Count();
```
Comment (DamienVStone) [inline API/Tenders.API.DAL.Postgres/Services/PostgresRepository.cs]: Why not use .LongCount() on Get() method?

File: API/Tenders.API/Startup.cs [hunk 3]
```diff
@@ -95,6 +98,13 @@ public void ConfigureServices(IServiceCollection services)
             {
                 c.SwaggerDoc("v1", new Info { Title = "Tenders API", Version = "v1" });
             });
+
+            var mapperConfig = new MapperConfiguration(cfg => {
```
Comment (DamienVStone) [inline API/Tenders.API/Startup.cs]: Move mapper registration to ServiceRegistrationPoint

File: API/Tenders.API.DAL.Postgres/Services/PostgresRepository.cs (hunk 2 above)
Comment (remmintan) [reply API/Tenders.API.DAL.Postgres/Services/PostgresRepository.cs]: Resolved: https://github.com/ssds-team/Tenders/pull/12/commits/9f09e82ea1990b798470882cbb4c89bd4017cd6d#diff-505e4db3ed38b144ff83ecae269cdba2R60-R64

**Pull request merged**